api.md

```diff
@@ -887,12 +888,51 @@ SELECT detach_tablespaces('conditions');
 ```
 
 ---
+## distributed_exec() [](distributed_exec)
+
+This command is used on an access node to execute a SQL command across
+the data nodes of a distributed database. This command can be used to,
+e.g., create cluster-wide objects across all instances of a
+distributed database. A common use case is creating the roles and
```

Comment:
I prefer to not use words "common", "often", and similar subjective words in formal texts since it can give the wrong impression.